server/handlers/meshery_application_handler.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/layer5io/meshery/server/meshes"
 	"github.com/layer5io/meshery/server/models"
 	pCore "github.com/layer5io/meshery/server/models/pattern/core"
+	"github.com/layer5io/meshkit/models/meshmodel"
```

Comment:
Where are we providing feedback to the client that X number of models and X number of components were registered?